docs/setup-cluster/deploy-cluster/firewall-rules.rst

```diff
@@ -6,12 +6,15 @@
  Firewall Rules
 ################
 
-The firewall rules must satisfy the following network access requirements for the master and agents.
+DRAFT The firewall rules must satisfy the following network access requirements for the master and
```

Comment:
https://hpe-aiatscale.slack.com/archives/CRYLQP7MW/p1696279343216229
* Compute nodes must be able to connect to master node on master's configured port.
* Compute nodes must be able to connect to each other on any port.
* Master node must be able to connect to compute nodes on any port.
* Compute nodes must be able to reach the docker image repository, or have the
  relevant images pre-downloaded.
* Compute nodes must have access to the desired checkpoint storage.
* Master node must also have access to desired checkpoint storage.
  * (this isn't technically required, but it's coming soon)
* Optionally, client nodes may have access to checkpoint storage for higher
  performance checkpoint access.
* Master node must have access to postgres
* Compute nodes must have access to network resources required by user tasks.
  Frequently this includes installing packages from pypi, for instance.
* Client machines must have access to the master node on master's configured port.